assets/js/src/form_editor/store/selectors.jsx

```diff
@@ -187,4 +187,18 @@ export default {
       return result;
     }, null);
   },
+  getEditorHistoryOffset(state) {
+    return state.editorHistoryOffset;
+  },
+  hasEditorUndo(state) {
+    let offset = state.editorHistoryOffset;
+    if (state.editorHistory.length > 1) {
```

Comment:
Do you think this would deserve a comment? It took me awhile to understand why do we need to add 1.